src/bo4e/com/zeitraum.py

```diff
@@ -36,9 +32,13 @@ class Zeitraum(COM):
 
     """
 
-    einheit: Optional["Mengeneinheit"] = None
-    dauer: Optional[Decimal] = None
-    startdatum: Optional[pydantic.AwareDatetime] = None
-    enddatum: Optional[pydantic.AwareDatetime] = None
-    startzeitpunkt: Optional[pydantic.AwareDatetime] = None
-    endzeitpunkt: Optional[pydantic.AwareDatetime] = None
+    startdatum: Optional[date] = None
+    """Startdatum, inklusiv"""
+    enddatum: Optional[date] = None
+    """Enddatum, inklusiv"""
+    startuhrzeit: Optional[time] = None
+    """Startuhrzeit, inklusiv mit Zeitzone"""
+    enduhrzeit: Optional[time] = None
+    """Enduhrzeit, exklusiv mit Zeitzone"""
+    dauer: Optional[str] = None
+    """Dauer"""
```

Comment:
Hier würde ich in den Docstring noch das wie im Issue erwähnen, dass es über die Norm gelöst wird